src/jdk.internal.jvmstat/share/classes/sun/jvmstat/monitor/HostIdentifier.java

```diff
@@ -343,7 +343,7 @@ public VmIdentifier resolve(VmIdentifier vmid)
         String host = vmid.getHost();
         String authority = vmid.getAuthority();
 
-        if ((scheme != null) && (scheme.compareTo("file") == 0)) {
+        if ("file".equals(scheme)) {
```

Comment:
Nit: Not your issue but you can also get rid of unneeded brackets ().

Reply:
I'm not sure. Which brackets do mean here?